pkg/router/route.go

```diff
@@ -34,6 +34,7 @@ func ConfigureEcho(allRoutes bool) *echo.Echo {
 		Skipper:         config.SkipLogging,
 	}))
 	e.Use(middleware.EnforceJSONContentType)
+	e.Use(middleware.EnforceOrgId)
```

Comment:
I like the use of middleware to solve this problem!   looking at the implementation, i noticed you are parsing the identity header manually, but this shouldn't actually be needed, as something like this *should* work, which pulls it from the context:

```		xRHID := identity.Get(c.Request().Context())```

However it didn't.  I poked around, and this middleware call should add it to the context:
```
	e.Use(middleware.WrapMiddlewareWithSkipper(identity.EnforceIdentity, middleware.SkipAuth))
```
I think if you move your `e.Use(middleware.EnforceOrgId)` after this line it should work and be much simpler.

But to be honest, i think your line here is in the right place but `	e.Use(middleware.WrapMiddlewareWithSkipper(identity.EnforceIdentity, middleware.SkipAuth))
` is in the wrong place.  So rather, i'd move that up above your line.

Reply:
ok awesome, that does simplify it quite a bit. thank you! so should i remove `e.Use(middleware.WrapMiddlewareWithSkipper(identity.EnforceIdentity, middleware.SkipAuth))` from `ConfigureEchoWithMetrics` and just have it in the `ConfigureEcho` function? 

i also am seeing the response returns a `status=200` when switching the order of middlewares for some reason, like this:

`ERR ../../../go/pkg/mod/github.com/ziflex/lecho/v3@v3.5.0/middleware.go:165 > error="code=403, message=Forbidden" bytes_in=0 bytes_out=0 host=localhost:8000 latency=0.296803 latency_human="296.803µs" method=GET referer= remote_ip=::1 request_id=XVOakqkgRgpJBCBPAUerXiilEjMFJWIF status=200 uri=/api/content-sources/v1.0/repositories/ user_agent=PostmanRuntime/7.35.0`

adding `c.Error(err)` to the EnforceOrgId middleware function forces it to return a 403 but i'm not sure if this is the best way to handle that?